pkg/storage/postgres/query.go

```diff
@@ -465,3 +465,19 @@ func (c *Client) TaskRuns(
 
 	return nil
 }
+
+func (c *Client) ReleaseExists(ctx context.Context, name string) (bool, error) {
+	const query = `
```

Comment:
Not using `COUNT` might perform better:

```
SELECT EXISTS(SELECT 1 FROM release WHERE name = $1)
```

Reply:
The current query also has `LIMIT 1`.
I expect that the difference will be barely measurable, seems not worth to me to measure, compare them + eventually adapt the code.

Reply:
LIMIT is notoriously bad at optimization, it also doesn't make sense to use LIMIT here since `name` should be unique. Returning more than 1 row should be detected as an error.